sdks/python/apache_beam/io/filesystemio.py

```diff
@@ -75,7 +75,7 @@ def finish(self):
     File should be fully written upon return from this method.
 
     Raises:
-      Any error encountered during the upload.
+      :Any error encountered during the upload.
```

Comment:
Is the added colon necessary? It looks out of place;
https://github.com/apache/beam-site/blob/bc0304ff55a23a984f108cf3355fc5c5fadb7148/pydoc/2.19.0_generated_in_py3/_modules/apache_beam/io/filesystemio.html#L248

Reply:
Confirmed it's unnecessary. removed.